src/lucky/routeable.cr

```diff
@@ -19,6 +19,10 @@ module Lucky::Routeable
     # **See also** our guides for more information and examples:
     # * [Routing](https://luckyframework.org/guides/actions-and-routing/#routing)
     macro {{ http_method.id }}(path)
+      \{% unless path.starts_with?("/") %}
+        \{% path.raise "Paths must start with a slash. For example, '" + path + "' is not a valid path, but '/" + path + "' is." %}
```

Comment:
I think adding the example will definitely help. I think it could be shortened to this: `"Path must start with a slash. Example: \"/#{path}\""`. Also note the typo Paths -> Path at the beginning